membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -103,6 +100,73 @@ class AccountController(commonActions: CommonActions, override val controllerCom
     }
   }
 
+  private def updateDD[P <: SubscriptionPlan.AnyPlan : SubPlanReads] = BackendFromCookieAction.async { implicit request =>
+    // TODO - refactor to use the Zuora-only based lookup, like in AttributeController.pickAttributes - https://trello.com/c/RlESb8jG
```

Comment:
I think it's probably worth taking this approach for the initial implementation rather than adding this as a todo

Reply:
this TODO is copied from the credit card one - I could try this approach now but based on wider discussion it looks likely we'll need to refactor all methods like this in the near future

Reply:
i'm happy to do this now if you still think it's worth it though?

Reply:
The refactor to use sub ID is in the current sprint for me and Jonny so this TODO will be removed before the year is out.